tests/cms/views/contacts/test_contact_utils.py

```diff
@@ -0,0 +1,224 @@
+    "json": {
+        "url": "/augsburg/contact/3/",
+        "name": "Integrationsberatung: Mariana Musterfrau (mariana-musterfrau@example.com, +49 (0) 123456789, https://integreat-app.de/)",
+        "details": {
+            "address": "show address",
+            "area_of_responsibility": "show area of responsibility",
+            "name": "show name",
+            "email": "show email",
+            "phone_number": "show phone number",
+            "website": "show website",
+        },
+    },
+    "details": "name,email",
+    "card": '<div><div contenteditable="false" data-contact-id="3" data-contact-url="/augsburg/contact/3/?details=name,email" class="contact-card notranslate" dir="ltr" translate="no"><a href="/augsburg/contact/3/?details=name,email" style="display: none">Contact</a><h4>Mariana Musterfrau</h4><p><img src="http://localhost:8000/static/svg/email.svg" alt="Email: " width="15" height="15"/><a href="mailto:mariana-musterfrau@example.com">mariana-musterfrau@example.com</a></p></div></div>',
+    "stub": '<div><div data-contact-id="3" data-contact-url="/augsburg/contact/3/?details=name,email"></div></div>',
+}
+
+CONTACT2 = {
+    "json": {
+        "url": "/augsburg/contact/4/",
+        "name": "(generalcontactinformation@example.com, +49 (0) 123456789, https://integreat-app.de/)",
+        "details": {
+            "address": "show address",
+            "email": "show email",
+            "phone_number": "show phone number",
+            "website": "show website",
+        },
+    },
+    "card": '<div><div contenteditable="false" data-contact-id="4" data-contact-url="/augsburg/contact/4/?details=phone_number" class="contact-card notranslate" dir="ltr" translate="no"><a href="/augsburg/contact/4/?details=phone_number" style="display: none">Contact</a><p><img src="http://localhost:8000/static/svg/call.svg" alt="Phone Number: " width="15" height="15"><a href="tel:+49123456789">+49 (0) 123456789</a></p></div></div>',
+    "stub": '<div><div data-contact-id="4" data-contact-url="/augsburg/contact/4/?details=phone_number"></div></div>',
+}
+
+
+def strip(arg: str | object) -> str:
+    return re.sub(r"\s+", "", str(arg), flags=re.UNICODE)
+
+
+@pytest.mark.django_db
+def test_search_contact_single(
+    load_test_data: None,
+    login_role_user: tuple[Client, str],
+    settings: SettingsWrapper,
+) -> None:
+    """
+    Test that querying for a specific contact returns that contact to authorized users
+    """
+    settings.LANGUAGE_CODE = "en"
+    client, role = login_role_user
+
+    response = client.post(
+        reverse(
+            "search_contact_ajax",
+            kwargs={
+                "region_slug": REGION_SLUG,
+            },
+        ),
+        json.dumps(
+            {
+                "query_string": "Mariana Musterfrau",
+            }
+        ),
+        content_type="application/json",
+    )
+
+    # to do: needs an update after.
+    # https://github.com/digitalfabrik/integreat-cms/issues/2958
```

Comment:
Thank you for highlighting this!
I think we can merge your PR first and I'll adjust this line in my PR for the permissions :)

Reply:
Alright, sounds good!

Reply:
And I forgot to do so. That's why we have the failing tests!